Documentation/how-to-create-postman-collections.md

```diff
@@ -0,0 +1,86 @@
+# How to create Postman collections
+
+We use [Postman](https://www.getpostman.com/) heavily at Nodes for tasks like:
+
+- Calling our internal and external APIs to see if they behave as expected
+- Documenting our APIs
+- Writing tests for our APIs
+- Mocking our APIs
+- Setting up monitoring for our APIs
+
+To align how we work with Postman, this document was created.
+
+## How to setup a new collection
+
+### Creating a collection
+
+Each project at Nodes that contains an API should have a corresponding Postman collection. A collection can be considered as a project or simply a folder in Postman. The collection should have the same name as  the project on GitHub but without the framework suffix. See [this document](https://github.com/nodes-projects/readme/blob/master/general/new-repository.md) for more info on our repo naming convention (internal link).
+
+### Sharing a collection
+
+With your new collection created, the next step is to make sure the rest of your team is able to see it. To do this, you're going to click the "…" icon on your new collection and click "Share Collection". This will bring up a new dialogue where you're able to specify how to share the collection:
+
+- Team: Can View
+- Individual permissions for each backend developer at Nodes: Can Edit
+
+This will allow consumers of the API to read about it as well as trying it out. It will also allow any other backend developer who might work on the project in the future to edit the collection if needed. Remember that you need to be invited to the Nodes team on Postman before you can publish it to the team.
+
+### Structure of a collection
+
+At the top level, the folders created should follow the different domains of the available endpoints. These domains will most likely follow the different models and controllers in your project. As an example, let's consider an API for interacting with a blog. At the top level, the collection could have the following folders:
+
+- Posts
+- Authors
+- Categories
+
+Sometimes, it makes sense to have nested folders to group related endpoints within a specific domain. In the above example, that could be categories for a specific post (e..g `/posts/:id/categories`). In that case, it's fine to have another folder inside Posts that group the endpoints related to categories for a specific post. Remember to keep the top level folder for Categories for any endpoints that are not tied to a specific post.
+
+Endpoints for the different domains should then be created within each top level domain folders. The endpoints should be named to describe their action. Having a reference to the domain in the name of the endpoint can be convenient when having multiple tabs open in the Postman application. Expanding the Posts folder could then have endpoints like:
```

Comment:
Loose the plural s on folders maybe:

> Endpoints for the different domains should then be created within each top level domain folder

And maybe a new line after:

> multiple tabs open in the Postman application.

So you end out with

> multiple tabs open in the Postman application.
>
> Expanding the Posts folder could then have endpoints like: